src/components/Header/Header.tsx

```diff
@@ -0,0 +1,74 @@
+import { Component, ReactNode } from 'react';
+import soundtrack from '/audio/zvezdnye-vojny-enikin-i-padme_(zzz.fm).mp3';
+import style from './_header.module.scss';
+import Logo from '/assets/images/star-wars-logo-png-image.png';
+import ErroLogo from '/assets/images/EternalGalacticEmpireLogo.webp';
+import { HeaderState } from '../../type/type';
+
+export class Header extends Component {
+  audio: HTMLAudioElement | null = null;
+  state: HeaderState = {
+    isPlaying: false,
+    hasError: false,
+  };
+
+  public componentDidMount(): void {
+    this.initializeAudio();
+  }
+  private initializeAudio(): void {
+    this.audio = new Audio(soundtrack);
+    this.audio.volume = 0.2;
+  }
+
+  private toggleMusic(): void {
+    if (this.audio) {
+      if (this.state.isPlaying) {
+        this.audio.pause();
+        this.setState({
+          isPlaying: false,
+        });
+      } else {
+        this.audio.play();
+        this.setState({
+          isPlaying: true,
+        });
+      }
+    }
+  }
```

Comment:
it improves readability of code
```suggestion
  private toggleMusic(): void {
    if (!this.audio) {
      return;
    }
      if (this.state.isPlaying) {
        this.audio.pause();
        this.setState({
          isPlaying: false,
        });
      } else {
        this.audio.play();
        this.setState({
          isPlaying: true,
        });
      }
    }
```